src/engine/SCons/Environment.xml

```diff
@@ -937,19 +937,28 @@ for a single special-case build.
 </para>
 
 <para>
-As a special case, the
-<varname>source_scanner</varname>
-keyword argument can
+&b-Command; builder accepts
+<varname>source_scanner</varname>,
+<varname>target_scanner</varname>,
+<varname>source_factory</varname>, and
+<varname>target_factory</varname>
+keyword arguments. This can
 be used to specify
 a Scanner object
-that will be used to scan the sources.
-(The global
+that will be used to scan node
```

Comment:
Why change from sources to node? That's less easily understandable.
Liekly you want separate paragraph for *_scanner than for *_factory. as they do different things.

Reply:
Let me make a new pass on the text

Reply:
@bdbaddog I updated the docs.. should be better than before